build/index.html

```diff
@@ -5,7 +5,7 @@
     <meta http-equiv="X-UA-Compatible" content="IE=edge" />
     <meta name="viewport" content="width=device-width, initial-scale=1.0" />
     <title>Covid Sentinel</title>
-  <script defer src="bundle.js"></script></head>
+  <script defer src="bundle.js"></script><link href="styles.css" rel="stylesheet"></head>
```

Comment:
Remember to create your branchs always from dev uptated from origin. This change should not be in here!!